Example/__tests__/Animation.test.js

```diff
@@ -0,0 +1,167 @@
+import React from 'react';
+import { View, Button } from 'react-native';
+import { render, fireEvent } from '@testing-library/react-native';
+import { renderHook } from '@testing-library/react-hooks';
+import Animated, {
+  useSharedValue,
+  useAnimatedStyle,
+  withTiming,
+  JestReanimated,
+} from 'react-native-reanimated';
+const {
+  withReanimatedTimer,
+  moveAnimationByTime,
+  moveAnimationByFrame,
+  getAnimatedStyle,
+} = JestReanimated;
+
+export default function TestComponent1(props) {
```

Comment:
I'm not fond of this name, I'm thinking about something like `<AnimatedStyleTestComponent />`. It's also the only component here so it doesn't make sense to number it.